src/infrastructure/components/Select/InputSelect.tsx

```diff
@@ -27,6 +38,7 @@ export const InputSelect = ({ className, label = '', name, error, register, opti
 
 interface InputSelectProps {
   options: Option[];
+  alpha3Code?: boolean;
```

Comment:
also, can we rename this prop as isAlpha3Code? just to keep our convention

Reply:
Of course. Renamed!